xfields/fieldmaps/interpolated_src/tricubic_coefficients.h

```diff
@@ -0,0 +1,1013 @@
+#ifndef XFIELDS_TRICUBIC_COEFFICIENTS_H
+#define XFIELDS_TRICUBIC_COEFFICIENTS_H
+
+void TriCubicInterpolatedFieldMap_construct_coefficients(
+    const double* b, double* coefs){
+
+    const double tri_consts[9] = {2.0, 3.0, 4.0, 6.0, 8.0, 9.0, 12.0, 18.0, 27.0};
+
```

Comment:
I guess these coefficients are autogenerated. This should be mentioned in the comments and the code used to generate the coefficients should be added somewhere in the package